common-content/en/blocks/evaluate/index.md

```diff
@@ -0,0 +1,33 @@
++++
+title="Evaluate ✏️"
+headless="true"
+time= 60
+[objectives]
+    1="Evaluate your current progress against previous modules"
++++
+
+It is essential to continually check in with yourself and evaluate your progress as you move through the course.
+
+One of the best ways of checking your progress against your past self is to go back to and tackle a problem from a previous part of the course.
+
+{{<note title="Pair up and check progress" type="activity">}}
+
+## Setup 🧰
+
+1. Trainees need to split up into pairs
+1. In your pairs, nominate a driver and a navigator. A driver will type out the code. A navigator will use the prep to steer you in your solution.
+
+## Activity
+
+1. Go back to JS1 Week 4 prep and familiarise yourself with the problem/function in that week
```

Comment:
You could decouple this further by just creating this as a general block. Instead of saying go back to JS1 W4, could you say go back 2 modules and find a problem?

Reply:
I'm connecting this to the spaced repetition block I made. (Look in HTML-CSS)

You could even pair it with one of those

Reply:
I did think about the generality here: ideally I wanted to be able to pass a reference to a particular problem as an argument but I think that saying 2 modules ago will be cool too